### DISCUSSION TITLE
exact match for levenshtein

### DISCUSSION THREAD

**Discussion by msiemionCalistapw** [moj-analytical-services/splink#741] [Q&A] [answered]
Why is levenshtein in my blocking rules at threshold greater than 75% ignoring exact matches in pairwise and is there a way around this?

**Comment by RobinL** [comment, marked as answer]
Please could you post a reproducible example, or at minimum the code you're running, so that we can help you

**Reply by msiemionCalistapw** [reply]
Sure! here's an example of my code and for some reason when dob and description for 2 rows are exactly matching it doesn't even consider the most-likely-to-be-clustered rows as pairs to compare.


settings = {
    "probability_two_random_records_match": 1/5,
    "link_type": "dedupe_only",
    "blocking_rules_to_generate_predictions": [
        """
        (l.dob = r.dob AND levenshtein(l.description, r.description) > 0.3)
        OR
        (l.dob = r.dob AND levenshtein(l.place, r.place) > 0.3 AND levenshtein(l.description, r.description) > 0.3)

        """,
    ],
    
    "comparisons": [
        description_comparison,
        place_comparison

    ],
}

**Reply by RobinL** [reply]
levenshtein is 0 for an exact match and then greater the more dissimilar its arguments.  It outputs integer values.

**Reply by msiemionCalistapw** [reply]
thank you very much!
